tests/phpunit/integration/Services/LabkiPageRegistryTest.php

```diff
@@ -179,17 +179,17 @@ public function testGetPageByTitle_FindsPage(): void {
 			'page_namespace' => 0,
 		] );
 		
-		$page = $registry->getPageByTitle( 'Unique Title For Search' );
+		$page = $registry->getPagebyFinalTItle( 'Unique Title For Search' );
```

Comment:
this test seems to test incorrect behavior? the `final_title` should be the page name in the wiki, and yet this test ensures that the wiki page does *not* match "final title"